include/libcyphal/runnable.hpp

```diff
@@ -14,10 +14,17 @@ namespace libcyphal
 class IRunnable
 {
 public:
+    virtual ~IRunnable() = default;
+
     virtual void run(const TimePoint now) = 0;
 
 protected:
-    virtual ~IRunnable() = default;
+    IRunnable()                 = default;
+    IRunnable(const IRunnable&) = default;
+    IRunnable(IRunnable&&)      = default;
+
+    IRunnable& operator=(const IRunnable&) = default;
+    IRunnable& operator=(IRunnable&&)      = default;
```

Comment:
I think runnables must be non-copyable, otherwise things like `AggregateRunnable` et all will break, because we have to be able to keep valid references to the interface. If the implementation object is moved, references lose validity.